tools/install.sh

```diff
@@ -188,35 +189,42 @@ ask_config() {
 }
 
 write_config_file() {
-    cat >${1} <<EOT
+    local config_file=${1:-${LOCAL_CONFIG_FILE:-default}}
```

Comment:
I'm not sure what this is for. Why is the fallback value "default"? It also seems that `config_file` is not used anyway.

Reply:
thanks for the feedback, see the new commit